### PR TITLE
user/massren: new package

### PR DESCRIPTION
**Pull request by subnut**

### CODE REVIEW

**Comment by subnut** [comment]
Seems like the CI build is failing due to some `proxy.golang.org` TCP error.
Force-pushing to rebuild.

**Comment by subnut** [comment]
Yeah no, still broken
```
0:00:05.604 => massren-1.5.7-r0: golang: go build -p 4 -v -trimpath -tags libsqlite3 -o /builddir/massren-1.5.7/build/
go: downloading github.com/jessevdk/go-flags v1.6.1
go: downloading github.com/kr/text v0.2.0
go: downloading github.com/mattn/go-sqlite3 v1.14.24
main.go:19:2: github.com/jessevdk/go-flags@v1.6.1: Get "https://proxy.golang.org/github.com/jessevdk/go-flags/@v/v1.6.1.zip": dial tcp: lookup proxy.golang.org on 127.0.0.53:53: read udp 127.0.0.1:34360->127.0.0.53:53: read: connection refused
main.go:20:2: github.com/kr/text@v0.2.0: Get "https://proxy.golang.org/github.com/kr/text/@v/v0.2.0.zip": dial tcp: lookup proxy.golang.org on 127.0.0.53:53: read udp 127.0.0.1:34360->127.0.0.53:53: read: connection refused
profile.go:12:2: github.com/mattn/go-sqlite3@v1.14.24: Get "https://proxy.golang.org/github.com/mattn/go-sqlite3/@v/v1.14.24.zip": dial tcp: lookup proxy.golang.org on 127.0.0.53:53: read udp 127.0.0.1:34360->127.0.0.53:53: read: connection refused
```

File: user/massren/template.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "7e7dd149bd3364235247268cc684b5a35badd9bee22f39960e075c792d037a8c"
+hardening = ["vis"]
+
+
+def post_extract(self):
+    self.rm("vendor", True)
+
+
+def post_install(self):
```
Comment (q66) [inline user/massren/template.py:22]: just !distlicense it

File: user/massren/template.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+makedepends = ["sqlite-devel"]
+go_build_tags = ["libsqlite3"]
+go_check_tags = ["libsqlite3"]
+pkgdesc = "Rename multiple files using any text editor"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "MIT"
+url = "https://github.com/laurent22/massren"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "7e7dd149bd3364235247268cc684b5a35badd9bee22f39960e075c792d037a8c"
+hardening = ["vis"]
```
Comment (q66) [inline user/massren/template.py:15]: this is useless

File: user/massren/template.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def post_extract(self):
+    self.rm("vendor", True)
```
Comment (q66) [inline user/massren/template.py:19]: use a named argument like everything else

**Pull request merged**